canonical-web-and-design/landscape-brochure.internal
# Initial site structure and stub files for templates

Added the initial structure of the site - copied from jaas.ai

## QA
 - ./run
 - Go to the [homepage](http://0.0.0.0:8000) and check that the site renders
